conda/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set data= load_setup_py_data() %}
+package: 
+  name: ts-integrationtests
+  version: {{ data.get('version') }}
+
+source:
+  path: ..
+
+build:
+  noarch: generic
+  script: python -m pip install --ignore-installed -e .
```

Comment:
Take a look at [this confluence page](https://confluence.lsstcorp.org/x/0Il-Cw), it has some updates to the conda recipe to make them forward compatible with different versions of python..